docs/developing/installing/targets.rst

```diff
@@ -0,0 +1,3 @@
+.. _Git repo named h: https://github.com/hypothesis/h/
```

Comment:
This indirection to me seems overly complicated versus just duplicating the links in each document...